openmc/data/product.py

```diff
@@ -0,0 +1,187 @@
+from collections.abc import Iterable
+from io import StringIO
+from numbers import Real
+import sys
+
+import numpy as np
+
+import openmc.checkvalue as cv
+from openmc.mixin import EqualityMixin
+from .angle_energy import AngleEnergy
+from .function import Tabulated1D, Polynomial, Function1D
+
+
+class Product(EqualityMixin):
+    """Secondary particle emitted in a nuclear reaction
+
+    Parameters
+    ----------
+    particle : str, optional
+        What particle the reaction product is. Defaults to 'neutron'.
```

Comment:
nit: May read better as: `The particle type of the reaction product. Defaults to 'neutron'.`